packages/components/src/base-control/index.js

```diff
@@ -7,31 +7,32 @@ import classnames from 'classnames';
  * Internal dependencies
  */
 import VisuallyHidden from '../visually-hidden';
+import { Control, Field, Label, Help } from './index.styles';
 
 function BaseControl( { id, label, hideLabelFromVision, help, className, children } ) {
 	return (
-		<div className={ classnames( 'components-base-control', className ) }>
-			<div className="components-base-control__field">
+		<Control className={ classnames( 'components-base-control', className ) }>
```

Comment:
Keeping the "unused" classnames for backwards compatibility (they're also used by components that compose this component).